ui/src/admin/containers/AdminPage.js

```diff
@@ -27,6 +41,33 @@ class AdminPage extends Component {
     }
   }
 
+  handleClickCreate(type) {
```

Comment:
Adds user to state only. It's written to be able to be used for adding a role as well, and will only work with those types. The proposed guard clause would be fine if it's just dedicated to "users", but that's not how it's currently intended.